examples/shader/specialized_mesh_pipeline.rs

```diff
@@ -427,6 +427,7 @@ fn queue_custom_mesh_pipeline(
                     asset_id: AssetId::<Mesh>::invalid().untyped(),
                 },
                 (render_entity, visible_entity),
+                mesh_instance.current_uniform_index,
```

Comment:
I think this is the same as `input_index` provided by `MeshPipeline::get_binned_index` above, which raises the bigger question of whether we could remove the `get_binned_index` method entirely, as it's only being used for unbatchables atm, which may make implementing it confusing. Minimally, the comment on `get_binned_index` should be changed to note it's not used for binning just unbatchables.

Reply:
Good point, I changed the comment.